content/en/docs/migrating/_index.md

```diff
@@ -0,0 +1,835 @@
+	if err != nil {
+		// handle error
+	}
+	totalObjects += len(output.Contents)
+}
+fmt.Println("total objects:", totalObjects)
+```
+
+### Waiters
+
+Service operation waiters are no longer invoked as methods on the service client. To use a waiter you first construct
+the operations associated waiter type, and then invoke the desired wait condition method. For example,
+to wait for a {{% alias service=S3 %}} Bucket to exist must construct a waiter for the `HeadBucket` operation. Use the
+[s3.NewHeadBucketWaiter]({{< apiref "service/s3#NewHeadBucketWaiter" >}}) constructor to create a
+[s3.HeadBucketWaiter]({{< apiref "service/s3#HeadBucketWaiter" >}}). The `s3.HeadBucketWaiter` provides a
+`WaitUntilBucketExists` method which can be used to wait for a bucket to become available.
+
+## Features
+
+### {{% alias service=EC2 %}} Instance Metadata Service
+
+The {{% alias sdk-go %}} provides an {{% alias service=EC2 %}} Instance Metadata Service (IMDS) client that you can use
+to query the local IMDS when executing your application on an {{% alias service=EC2 %}} instance. The IMDS client is
+a separate Go module that can be added to your application by using `go get github.com/aws/aws-sdk-go-v2/ec2imds`. The
+client constructor and method operations have been updated to match the design of the other SDK service clients.
+
+#### Example
+
+##### V1
+
+````go
+import "github.com/aws/aws-sdk-go/aws/ec2metadata"
+
+// ...
+
+client := ec2metadata.New(sess)
+
+region, err := client.Region()
+if err != nil {
+	// handle error
+}
+````
+
+##### V2
+
+````go
+import "context"
+import "github.com/aws/aws-sdk-go-v2/ec2imds"
+
+// ...
+
+client := ec2imds.NewFromConfig(cfg)
+
+region, err := client.GetRegion(context.TODO())
+if err != nil {
+	// handle error
+}
+````
+
+### {{% alias service=S3 %}} Transfer Manager
+
+The {{% alias service=S3 %}} transfer manager is available for managing uploads and downloads of objects
+concurrently. This package is located in a Go module outside the service client import path. This module
+can be retrieved by using `go get github.com/aws/aws-sdk-go-v2/feature/s3/manager`.
+
+[s3.NewUploader]({{< apiref v1="service/s3/s3manager#NewUploader" >}}) and
+[s3.NewUploaderWithClient]({{< apiref v1="service/s3/s3manager#NewUploaderWithClient" >}}) have been replaced with the
+constructor method [manager.NewUploader]({{< apiref "feature/s3/manager#" >}}) for creating an Upload manager client.
+
+[s3.NewDownloader]({{< apiref v1="service/s3/s3manager#NewDownloader" >}}) and
+[s3.NewDownloaderWithClient]({{< apiref v1="service/s3/s3manager#NewDownloaderWithClient" >}}) have been replaced with a
+single constructor method [manager.NewUploader]({{< apiref "feature/s3/manager#NewDownloader" >}}) for creating a
+Dwonload manager client.
+
+### {{% alias service=CFlong %}} Signing Utilities
+
+The {{% alias service=CFlong %}} signing utilities are available for use in the {{% alias sdk-go %}}. This package is
```

Comment:
These two sentences seem awkward. Maybe swap the first around and combine them:

The {{% alias sdk-go %}} includes signing utilities in a Go module outside the service client import path.